directories_and_file_paths/directories_and_file_paths.md

```diff
@@ -26,35 +32,29 @@ After completion of this module, learners will be able to:
 good_first_module: false
 collection: infrastructure_and_technology
 coding_required: false
+@sets_you_up_for
 
-link:  https://storage.googleapis.com/chop-dbhi-arcus-education-website-assets/css/styles.css
-
-script: https://kit.fontawesome.com/83b2343bd4.js
-
--->
-
-# Directories and File Paths
-
-<div class = "overview">
-
-## Overview
-
-@comment
-
-**Is this module right for me?**
-
-@long_description
+- bash_command_line_101
+- bash_command_line_102
 
-**Estimated time to completion:** @estimated_time
+@end
 
-**Pre-requisites**: None.
+@depends_on_knowledge_available_in: 
 
-**Learning Objectives**:
+@end
+@version_history 
```

Comment:
Version history is not rendering correctly in Liascript for some reason-- but I'm not immediately sure why? The section is visible but the versions and descriptions I see in the raw markdown are different in the rendered page (it looks like sample text). I checked in Chrome and Safari.

Reply:
Oooh amazing, thanks for figuring this out!